### ISSUE TITLE
[Bug][Password Input] Regression : password input is not accessible anymore with VO or FKA

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#614]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone

### OS Version

26

### App version

1.0.0 (649)

### Describe the issue

Impossible to fill the password input with Voice Over or Full Keyboard Access
It works well on 1.0.0(625) for example

### Expected Behavior

Input shall be accessible

**Comment by nouha06** [comment]
New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (688)
Version: 1.3.0
Build type: ALPHA
Build details (GitHub): 614
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (688)
Version: 1.3.0
Build type: ALPHA
Build details (GitHub): 614
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by pya35** [comment]
fixed !
